farm/data_handler/data_silo.py

```diff
@@ -181,10 +181,12 @@ def _get_dataset(self, filename, dicts=None):
             if filename:
                 desc += f" {filename}"
             with tqdm(total=len(dicts), unit=' Dicts', desc=desc) as pbar:
-                for dataset, tensor_names in results:
+                for dataset, tensor_names, problematic_samples in results:
```

Comment:
Doesnt this statement break in case when the problematic samples are not returned?

Reply:
In this context, `results` is always the output of `_dataset_from_chunk( )` which I modified to always return 3 objects so I don't think this unpacking is an issue. As an asside, this function (i.e. `_get_dataset( )`) is the only method in our code base which calls `_dataset_from_chunk( )` so it shouldn't be problematic that I've modified it to return 3 objects.

Reply:
Ok, agreed, but this means we always need to return those ids. 
We do have a flag in dataset_from_dicts, return_problematic=True, which seems to break the processing when set to false, right?